docs/core/preview3/tools/csproj.md

```diff
@@ -25,81 +25,103 @@ see [the MSBuild project file](https://docs.microsoft.com/visualstudio/msbuild/m
 Specifies a NuGet dependency in the project. The `Include` attribute specifies the package ID. 
 
 ```xml
-<PackageReference Include="<package-id>">
-    <Version></Version>
-    <PrivateAssets></PrivateAssets>
-    <IncludeAssets></IncludeAssets>
-    <ExcludeAssets></ExcludeAssets>
-</PackageReference>
+<PackageReference Include="<package-id>" Version="" PrivateAssets="" IncludeAssets="" ExcludeAssets="" />
 ```
 
 #### Version
 `<Version>` specifies the version of the package to restore. The element respects the rules of the NuGet versioning scheme.
```

Comment:
Conventionally, attributes aren't bracketed -- 'Version' instead of '<Version>'. You've left them bracketed here throughout your changes.